File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int n = nums.length;
        for (int i=0;i<n;) {
            if (nums[i]==val) nums[i]=nums[--n];
            else ++i; 
        }
        return n;
    }
}

File: Data Structures & Algorithms/sliding-window-maximum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    maxSlidingWindow(nums, k) {
        let res = [];

        for (let i = 0; i <= nums.length - k; i++) {
            let localMax = nums[i];

            for (let j = i; j < i + k; ++j) {
                localMax = Math.max(localMax,nums[j]);
            }
            res.push(localMax);
        }
        return res;
    }
}
